client-v2/src/actions/__tests__/ArticleFragments.spec.ts

```diff
@@ -47,19 +49,19 @@ const root = (state: any = {}, action: any) => ({
 
 const buildStore = (added: ArticleFragmentSpec, collectionCap = Infinity) => {
   const groupA: ArticleFragmentSpec[] = [
-    ['a', '1', [['g', '7']]],
-    ['b', '2', undefined],
-    ['c', '3', undefined]
+    ['a', '1', [['g', '7']], {}],
```

Comment:
Minor: tuples can have optional elements on the right hand side, so we could introduce a question mark in the definition of ArticleFragmentSpec for the meta to make these new object literals redundant 👍